frontend/src/components/CardDetails/CardDetails.tsx

```diff
@@ -29,14 +28,14 @@ export function CardDetails({ productTitle, price, store }: CardDetailsProps) {
     return (
       <div data-testid="card-details">
         <RUICard.Header className="py-0">
-          <RUICard.Title>{store}</RUICard.Title>
+          <RUICard.Title>{supermarket}</RUICard.Title>
         </RUICard.Header>
         <RUICard.Content className="py-0">
-          <p>{productTitle}</p>
+          <p>{title}</p>
         </RUICard.Content>
         <RUICard.Content className="flex justify-around  items-center">
-          <p className="text-lg font-semibold">${price}</p>
-          <Button onClick={() => addToCart(productTitle)}>Add to cart</Button>
+          <p className="text-lg font-semibold">${price.value}</p>
+          <Button onClick={() => addToCart()}>Add to cart</Button>
```

Comment:
💅 if we're not passing anything into the function here this can just be: 

```suggestion
          <Button onClick={addToCart}>Add to cart</Button>
```